plugins/controlboard/include/ControlBoard.hh

```diff
@@ -2,6 +2,8 @@
 
 #include <ControlBoardData.hh>
 
+#include <cstddef>
+#include <gz/msgs/details/wrench.pb.h>
```

Comment:
Is this required? details tipically indicates private headers that are not meant to be included.

Reply:
Solved in Solved in https://github.com/robotology/gz-sim-yarp-plugins/pull/153/commits/fd8043fb3c1ec219c31f9d35cd441383ffcfa57b